src/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -0,0 +1,138 @@
+		const issues = logSingleWord("Hello", position);
+		expect(issues).toEqual(
+			{message: `${"Hello"} von Stelle ${position.fromIndex} bis ${position.toIndex} ist ein unerlaubtes Wort! \n`,
+		position: {
+			fromIndex: position.fromIndex,
+			toIndex: position.toIndex,
+			}});
+	});
+
+/**
+ * The next two testcases should never actually appear in practice. Since an empty word and a single whitespace
+ * are not classified as "words" for our purposes. For the sake of completeness those cases are
+ * tested nonetheless
+ */
+
+	it("Logs a empty word correctly", () => {
+		const position: Position = {fromIndex: 0, toIndex: 0};
+		const issues = logSingleWord("", position);
+		expect(issues).toEqual(
+			{message: `${""} von Stelle ${position.fromIndex} bis ${position.toIndex} ist ein unerlaubtes Wort! \n`,
+		position: {
+			fromIndex: position.fromIndex,
+			toIndex: position.toIndex,
+			}});
+	});
+
+	it("Logs a word consisting only of a whitespace correctly", () => {
+		const position: Position = {fromIndex: 0, toIndex: 1};
+		const issues = logSingleWord(" ", position);
+		expect(issues).toEqual(
+			{message: `${" "} von Stelle ${position.fromIndex} bis ${position.toIndex} ist ein unerlaubtes Wort! \n`,
+		position: {
+			fromIndex: position.fromIndex,
+			toIndex: position.toIndex,
+			}});
+	});
+});
+
+describe("CollectAllInvalidWordsInIssues", () => {
+	it("Creates a correct Issue-array for one wrong word containing only this word", () => {
+		const issues = collectInvalidWordsInIssues("WrongWord RightWord", ["WrongWord"]);
+		expect(issues).toEqual(
+			[{message: "WrongWord", position: {fromIndex: 0, toIndex: 9}}]);
+	});
+
+	it("Creates a correct Issue-array for a word consisting only of a Whitespace", () => {
+		const issues = collectInvalidWordsInIssues("Te st", [" "]);
+		expect(issues).toEqual(
+			[{message: " ", position: {fromIndex: 2, toIndex: 3}}]);
+	});
+});
+
+describe("getAllIssues", () => {
+	it("Creates a correct Issue-array for one wrong word containing only this word", () => {
+		const issues = getAllIssues("WrongWord RightWord");
+		expect(issues).toEqual(
+			[{message: "WrongWord", position: {fromIndex: 0, toIndex: 9}}]);
+	});
+
+	it("Creates a correct Issue-array for two words seperated by a Whitespace", () => {
+		const issues = getAllIssues("Te st");
+		expect(issues).toEqual(
+			[{message: "Te", position: {fromIndex: 0, toIndex: 2}}, {message: "st", position: {fromIndex: 3, toIndex: 5}}]);
+	});
+
+	it("Detects a String of word1.word2 as two words", () => {
+		const issues = getAllIssues("Te.st");
+		expect(issues).toEqual(
+			[{message: "Te", position: {fromIndex: 0, toIndex: 2}}, {message: "st", position: {fromIndex: 3, toIndex: 5}}]);
+	});
+
+	it("Detects a String of word1,;:<>=word2 as two words", () => {
+		const issues = getAllIssues("Te,;<>=st");
+		expect(issues).toEqual(
+			[{message: "Te", position: {fromIndex: 0, toIndex: 2}}, {message: "st", position: {fromIndex: 7, toIndex: 9}}]);
```

Comment:
Please leave the opening `[` on the same line as the opening `(` and put the `]);` on its own line. Add a linebreak after each array element and a comma after the last one.